src/components/History/History.tsx

```diff
@@ -62,13 +63,25 @@ const History: React.FC<HistoryProps> = ({
     return sortArrayByDate(history)
   }, [history])
 
+  useEffect(() => {
+    trackEvent('HL Chat History Viewed', {});
+  }, []);
```

Comment:
This will always call right now, because the component is always rendered but hidden unless you open history.

Reply:
Removed